Tensile/KernelWriterAssembly.py

```diff
@@ -7708,11 +7948,26 @@ def localReadDo(self, kernel, bufferIdx, iui, epsi, uIdx, tP):
             if kernel["MatrixInstruction"]:
               if not kernel["TransposeLDS"] or (kernel["ProblemType"]["TLU%s"%tP["tensorChar"]] and (kernel["TransposeLDS"] == 1)):
                 tIdx = tP["tensorIdx"]
-                readOffsetWidth = kernel["MacroTile%u" % tIdx] // kernel["ThreadTile%u" % tIdx]
+                #readOffsetWidth = kernel["MacroTile%u" % tIdx] // kernel["ThreadTile%u" % tIdx]
+                if tP["isA"]:
```

Comment:
Looks like this is duplicated from above

Reply:
cleaned up some part of code more clean-up in next version